### DISCUSSION TITLE
Feedback on visual style?

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [learn-awesome/learndb#13] [Show and tell]
<img width="1689" alt="image" src="https://user-images.githubusercontent.com/19304/171865328-b8c22482-15d7-476e-9b91-a1c482809f29.png">

<img width="1697" alt="image" src="https://user-images.githubusercontent.com/19304/171865412-e3bec0b3-3205-4f89-8d95-4c0af8476c7a.png">

<img width="1687" alt="image" src="https://user-images.githubusercontent.com/19304/171865546-9cd68586-a096-4754-a78f-9aaebe6164ae.png">

**Comment by MridulTi** [comment]
You guys are doing a pretty amazing job here but the thing is the page looks quite static with a constant navbar on the left it would be better if the nav bar was on top also shouldn't you guys be using React framework as you are already using JS it would be better if you added React elements in the page that would make the website more dynamic

**Reply by nileshtrivedi** [reply]
We decided to use Svelte framework which is more modern & productive than React. It still has components with props and state, but since it supports actual reactivity, code is much easier to write. [Do have a look](https://svelte.dev/examples/reactive-assignments).

About the visual design, there definitely is a scope for improvement. The reason we went with a vertical sidebar is because  laptops and desktops have widescreen aspect ratio, so vertical space is less available than horizontal. This way, we get more space available for actual content than navigation.

**Comment by nileshtrivedi** [comment]
New visual style is here (h/t @cssbubble) and I like this one much better:

<img width="1699" alt="image" src="https://user-images.githubusercontent.com/19304/173263135-a3a02843-93d0-4c9c-b84f-a42b4883ff66.png">

<img width="1694" alt="image" src="https://user-images.githubusercontent.com/19304/173263184-2b72e006-91f1-43d6-9bde-3d151272e177.png">

<img width="1682" alt="image" src="https://user-images.githubusercontent.com/19304/173263247-275ef166-ee59-4ba9-8123-f0e7e6d19988.png">

**Comment by beedell-roke** [comment]
> https://github.com/learn-awesome/learndb/discussions/13#discussion-4118072

Regardless of your visual style, I hope that you can incorporate the system theme like https://gitlab.com/coolercontrol/coolercontrol/-/issues/163#note_1717297642 demonstrates, not least for the sake of accessibility, but also consistency.

**Reply by nileshtrivedi** [reply]
Do you mean supporting [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) to switch between light mode and dark mode?
